dlup/experimental_backends/pyvips_backend.py

```diff
@@ -125,6 +129,25 @@ def _read_as_openslide(self, path: PathLike):
                 f"{path} does not have a parseable spacings property. You can overwrite it with `.mpp = (mpp_x, mpp_y)."
             )
 
+        if "openslide.bounds-x" in available_fields and "openslide.bounds-y" in available_fields:
+            offset = int(self._images[0].get("openslide.bounds-x")), int(self._images[0].get("openslide.bounds-y"))
+        else:
+            offset = (0, 0)
+
+        if "openslide.bounds-width" in available_fields and "openslide.bounds-height" in available_fields:
+            size = int(self._images[0].get("openslide.bounds-width")), int(
+                self._images[0].get("openslide.bounds-height")
+            )
+        else:
+            size = self._shapes[0]
+
+        self.__slide_bounds = offset, size
```

Comment:
why is this different than the one in the openslide backend?

Reply:
pyvips stores the parameters in a different location

Reply:
but wouldn't get `.get` method check if it exists and if not set a different value? justl ike you're doing in the above?

so no need for if-else statements and checking if the stuff is in available fields?

Reply:
It's not a dictionary, this is how pyvips works. If you call .get on a field that does not exist you get an error